.gitignore

```diff
@@ -9,4 +9,4 @@
 /www/doc
 /www/skin
 /vendor/
-/testmails
+/testmails
```

Comment:
You still have a modification here, you removed the tailing newline that was there before. :)